.github/workflows/application-onboarding-helm-template.yml

```diff
@@ -70,19 +70,22 @@ jobs:
 
       - name: Create application directory
         if: steps.directory_check.outputs.directory_exists == 'false'
-        run: mkdir -p vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}
+        run: mkdir -p vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/templates
 
-      - name: Copy Chart.yaml from template
-        if: steps.directory_check.outputs.directory_exists == 'false'
-        run: cp vsp-infra-helm/hello-world/Chart.yaml vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/Chart.yaml
+      - name: Copy hello-world
+        run: cp -ar vsp-infra-helm/hello-world/* vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}
 
-      - name: Modify Chart.yaml to have correct format
-        if: steps.directory_check.outputs.directory_exists == 'false'
-        run: cat <<< $(yq -e 'del(.keywords) | del(.sources) | .dependencies[0].name=.name | .dependencies[0].version=.version | .dependencies[0].repository="https://department-of-veterans-affairs.github.io/vsp-infra-helm"' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/Chart.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/Chart.yaml
+      # - name: Copy Chart.yaml from template
+      #   if: steps.directory_check.outputs.directory_exists == 'false'
+      #   run: cp vsp-infra-helm/hello-world/Chart.yaml vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/Chart.yaml
 
-      - name: Copy values.yaml from template
-        if: steps.directory_check.outputs.directory_exists == 'false'
-        run: cp vsp-infra-helm/hello-world/values.yaml vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
+      # - name: Modify Chart.yaml to have correct format
```

Comment:
Should we just remove the code that is commented out if its not needed anymore?

Reply:
That would be the smartest move. Whoops